src/GraphQL.ApiTests/GraphQL.MicrosoftDI.approved.txt

```diff
@@ -160,3 +160,12 @@ namespace GraphQL.MicrosoftDI
         public object GetService(System.Type serviceType) { }
     }
 }
+namespace GraphQL
+{
+    [System.AttributeUsage(System.AttributeTargets.Method | System.AttributeTargets.All)]
```

Comment:
Bump PublicApiGenerator to 10.3.0 to get rid of `| System.AttributeTargets.All`.

Reply:
Won’t we do that in a separate PR for the whole solution?

Reply:
ok